expression/builtin_time.go

```diff
@@ -2023,35 +2023,36 @@ type makeDateFunctionClass struct {
 }
 
 func (c *makeDateFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
-	sig := &builtinMakeDateSig{newBaseBuiltinFunc(args, ctx)}
+	tp := types.NewFieldType(mysql.TypeDate)
+	tp.Flen = 10
```

Comment:
@XuHuaiyu 
Thank you, I found it,and the MAX_DATE_WIDTH	is 10.
refer [https://github.com/mysql/mysql-server/blob/71f48ab393bce80a59e5a2e498cd1f46f6b43f9a/sql/sql_const.h#L59](https://github.com/mysql/mysql-server/blob/71f48ab393bce80a59e5a2e498cd1f46f6b43f9a/sql/sql_const.h#L59)